mne/_fiff/_digitization.py

```diff
@@ -186,7 +186,8 @@ def _read_dig_fif(fid, meas_info, *, return_ch_names=False):
                 dig.extend(tag.data)
             elif kind == FIFF.FIFF_MNE_COORD_FRAME:
                 tag = read_tag(fid, pos)
-                coord_frame = _coord_frame_named.get(int(tag.data.item()))
+                coord_frame = int(tag.data.item())
+                coord_frame = _coord_frame_named.get(coord_frame, coord_frame)
```

Comment:
I'm not following the purpose of `_coord_frame_named` here. AFAICT, it's a dict like `{key: key}` where each entry is one of `FIFF.FIFFV_(MNE_)COORD_*`.  prior to this change, the function of `_coord_frame_named` would have been to convert anything *not* in that dictionary to `None`. Now, IIUC it's basically going to be a no-op?

Reply:
There is a subtlety -- our `FIFF` constants subclass `int`. So the purpose of `_coord_frame_named` is actually meant to be to convert from a Python int to our int subclass *when possible*. So the bug here was not having `_coord_frame_named.get(<thing>, <thing>)`. In other words, the big bug was that we ever converted to `None` in the first place (we should just preserve the coord frame int if it's not in the dict).